packages/types/src/league.ts

```diff
@@ -1,10 +1,18 @@
 import type { Query } from "./db_client.js";
 import type { MongoDBItem, ObjectId, Sport } from "./utils.js";
-
+import type { AbilityLevel, User } from "@esp-group-one/types";
 interface BaseLeague extends MongoDBItem {
   name: string;
   ownerIds: ObjectId[];
   sport: Sport;
+  round: number;
+  people: User[];
```

Comment:
This is not required, nor do we want to be storing duplicate information. Each user stores the `leagues` which they are apart of